hs_core/hydroshare/resource.py

```diff
@@ -1046,8 +1046,53 @@ def publish_resource(user, pk, publish_type):
             doi = return_data['@id']
             resource.doi = doi
             resource.minid = ''
+            doi_url = 'https://ors.datacite.org/' + doi
             ident_md_args = {'name': 'doi',
-                       'url': 'https://ors.datacite.org/' + doi}
+                       'url': doi_url}
+
+    # register published resource in farishake
+    #retrieve an API Key to access FairShake registration API
+
+    request_data = {}
+    request_data['username'] = settings.FAIRSHAKE_USERID
+    request_data['password'] = settings.FAIRSHAKE_PASSWORD
+    response = requests.post(settings.FAIRSHAKE_URL +'/auth/login/',
+                            data=json.dumps(request_data),
+                            headers={"Content-Type": "application/json"})
+
+    if response.status_code != status.HTTP_200_OK:
+        logger.error("Error retrieving APIKey from FairShake API")
+        logger.error(response.status_code)
+        logger.error(response.text)
+        raise PublishException("This resource cannot be published because it has failed the FairShake registration process." + response.text)
+    else:
+        return_data = json.loads(response.content)
+        fairshake_apikey = return_data['key']
+
+        request_data = {}
+        request_data["title"] = resource.title
+        request_data["tags"] = "dcppc"
+        request_data["url"] =  resource_url
+        request_data["projects"] = [14]
+        request_data["rubrics"] = [11]
+
+        logger.info(json.dumps(request_data))
```

Comment:
remove this logger info or add a conditional to only show it when it is on DEBUG.